commit 350848717ee19b439ac51efd6d9dc2c88769ec3b
Author: Felix Yan <felixonmars@archlinux.org>
Date:   Mon Dec 12 22:52:53 2016 +0800

    Explicitly convert property_content to bytes
    
    property_content being an instance of `str` fails with Python 3.5:
    
    ```
      File "/usr/lib/python3.5/site-packages/dae/daeclient.py", line 81, in __init__
        self.window = Window(None, index_path)
      File "/usr/lib/python3.5/site-packages/dae/window.py", line 59, in __init__
        super(Window, self).__init__(parent if isDialog else None)
      File "/usr/lib/python3.5/site-packages/dae/shadowwindow.py", line 41, in __init__
        self.setShadowWidth(10)
      File "/usr/lib/python3.5/site-packages/dae/shadowwindow.py", line 60, in setShadowWidth
        xutils.show_shadow(xutils.get_xwindow(int(self.winId())), str(width))
      File "/usr/lib/python3.5/site-packages/dae/xutils.py", line 32, in show_shadow
        ewmh.setXWindowProperty(xwin, "DEEPIN_WINDOW_SHADOW", str(shadow_width))
      File "/usr/lib/python3.5/site-packages/dae/ewmh.py", line 119, in setXWindowProperty
        property_content,
      File "/usr/lib/python3.5/site-packages/Xlib/xobject/drawable.py", line 436, in change_property
        data = (format, data))
      File "/usr/lib/python3.5/site-packages/Xlib/protocol/rq.py", line 1338, in __init__
        self._binary = self._request.to_binary(*args, **keys)
      File "/usr/lib/python3.5/site-packages/Xlib/protocol/rq.py", line 1012, in to_binary
        v, l, fm = f.pack_value(field_args[f.name])
      File "/usr/lib/python3.5/site-packages/Xlib/protocol/rq.py", line 688, in pack_value
        data = array(array_unsigned_codes[size], val).tostring()
    TypeError: cannot use a str to initialize an array with typecode 'B'
    ```
    
    Change-Id: I3e1de284be81a5d0acc622f445364a181115958a

diff --git a/dae/ewmh.py b/dae/ewmh.py
index df1fc6b..207b93b 100644
--- a/dae/ewmh.py
+++ b/dae/ewmh.py
@@ -116,7 +116,7 @@ class EWMH:
             self.display.get_atom(property_type),
             Xatom.STRING,
             8,
-            property_content,
+            property_content.encode("ascii"),
             )
         self.display.sync()
 
